### DISCUSSION TITLE
Is there an option to remove auto-generated class?

### DISCUSSION THREAD

**Discussion by Emporea** [observablehq/plot#727] [Q&A] [answered]
To be able to completly set my own css style to plots I asked myself if its possible to remove the plot auto-generated class.
I know I can rename it via className and then remove it via element.classList.remove() but is there a more simple way?

**Comment by mbostock** [comment, marked as answer]
I am not aware of a more simple way. Plot requires a className in order to apply its stylesheet. We do not support disabling this stylesheet, but we do allow you to set whatever className you want as the top-level option (as you noted).

https://github.com/observablehq/plot/blob/5ce6cc0595b29839174b804c98f85ffbc4335770/src/plot.js#L83-L95
